libs/ui/src/hooks/smartcard_auth/use_dipped_smartcard_auth.ts

```diff
@@ -233,8 +234,11 @@ function useDippedSmartcardAuthBase({
     case 'logged_out':
       return {
         ...auth,
-        bootstrapAuthenticatedAdminSession: (electionHash: string) =>
-          dispatch({ type: 'bootstrap_admin_session', electionHash }),
+        bootstrapAuthenticatedElectionManagerSession: (electionHash: string) =>
+          dispatch({
+            type: 'bootstrap_election_manager_session',
+            electionHash,
+          }),
```

Comment:
Good q! Because I haven't fully removed the VVSG2 auth flows feature flag, just enabled it by default, it's still needed in the case of a rollback or someone testing locally with VVSG2 auth flows disabled (which is of course highly unlikely at this point). But for completeness/correctness, I'm only going to remove it once I fully remove the feature flag: https://github.com/votingworks/vxsuite/issues/2085